src/instructlab/lab.py

```diff
@@ -1041,19 +1043,31 @@ def train(
             )
             raise click.exceptions.Exit(1)
 
-    if not utils.is_macos_with_m_chip():
+    if backend == "pytorch":
         # Local
         from .llamacpp.llamacpp_convert_to_gguf import convert_llama_to_gguf
-        from .train.linux_train import linux_train
+        from .train.pytorch_train import pytorch_train
 
-        linux_train(
-            ctx=ctx,
+        if device.type == "mps":
+            if (
+                "PYTORCH_MPS_HIGH_WATERMARK_RATIO" not in os.environ
+                or "PYTORCH_ENABLE_MPS_FALLBACK" not in os.environ
+            ):
+                click.secho(
+                    "To use MPS, PYTORCH_MPS_HIGH_WATERMARK_RATIO must be set to 0.0 and PYTORCH_ENABLE_MPS_FALLBACK must be set to 1",
```

Comment:
What if there are set but don't have the expected values?

Reply:
If the user knows what they are doing enogh to set these, they probably have good reason for it :) 

I think the fallback will error out as it can only be 1 or 0.

The ratio will work as long as its set, but will be worse the further you get from 0.0

Reply:
Thanks for the clarification!